java/app/grapheneos/setupwizard/view/activity/WelcomeActivity.kt

```diff
@@ -25,6 +29,11 @@ class WelcomeActivity : SetupWizardActivity(R.layout.activity_welcome) {
     private lateinit var next: View
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        if (WizardManagerHelper.isUserSetupComplete(this)) {
+            superOnCreateAtBaseClass(savedInstanceState)
```

Comment:
just trying to understand under what circumstances will this happen?
setup wizard disables itself upon completion.